.gitignore

```diff
@@ -1,15 +1,96 @@
+# Created by https://www.gitignore.io/api/osx,intellij,android,gradle,java
+
+### OSX ###
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+
+### Intellij ###
+.idea/
+*.iml
+
+### Android ###
+# Built application files
+*.apk
+*.ap_
+
+# Files for the Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Gradle files
+.gradle/
+build/
+buildSrc/
```

Comment:
Out of curiosity, why ignore `buildSrc`? That's usually important.

Reply:
https://github.com/ben-manes/gradle-versions-plugin/pull/605.